.travis.yml

```diff
@@ -1,20 +1,26 @@
 language: node_js
 
-before_script:
- - sh -e /etc/init.d/xvfb start
-
-install: ant
-
 env:
- - DISPLAY=':99.0'
+  global:
+    - DISPLAY=':99.0'
+  matrix:
+    - MOZDOWNLOADPARAMS='-t daily'
+    - MOZDOWNLOADPARAMS='-t daily --branch=mozilla-aurora'
+    - MOZDOWNLOADPARAMS='-t release -v latest-beta'
+    - MOZDOWNLOADPARAMS='-t release -v latest'
+    - MOZDOWNLOADPARAMS='-t release -v latest-esr'
 
-script: ant test
+before_install:
+  - sudo apt-get update -qq
 
-notifications:
-  irc:
-    channels:
-      - "irc.mozilla.org#automation"
-    on_success: always
-    on_failure: always
-    use_notice: true
+install:
+  - sudo apt-get install -y ant python-pip
+  - sudo pip install mozdownload
+
+before_script:
+  - sh -e /etc/init.d/xvfb start
+  - mozdownload $MOZDOWNLOADPARAMS
+  - tar -xf *firefox*.tar.bz2
```

Comment:
Is the workspace being cleared between each build as listed in the matrix? If not we probably use the same build all the time.